#!/usr/bin/python3
def raise_exception_msg(message=""):
    raise NameError(message)

# On lève l'exception de la variable message à l'aide de NameError
